### ISSUE TITLE
Is there a way to make the header scrollable?

### ISSUE THREAD

**Issue by tgwow** [PedroBern/react-native-collapsible-segmented-view#6]
My header takes up almost the entire screen, it would be great if I could scroll it too. I tried to put in a ScrollView, but no success.

**Comment by PedroBern** [comment]
Yes, see https://github.com/PedroBern/react-native-collapsible-tab-view#scroll-on-header
